utils/provided-goi-analysis-functions.R

```diff
@@ -189,7 +188,23 @@ plot_goi_expression_sina <- function(normalized_sce,
       y = gene_expression,
       color = gene_expression)) +
       ggforce::geom_sina() +
-      theme(axis.text.x = element_text(angle = 90))
+      theme(axis.text.x = element_text(angle = 90)) +
+      stat_summary(
+        aes(group = !!gene_symbol_column_sym),
+        color = "red",
+        # median and quartiles for point range
+        fun = "mean",
+        fun.min = function(x) {
+          quantile(x, 0.25)
+        },
+        fun.max = function(x) {
+          quantile(x, 0.75)
+        },
+        geom = "pointrange",
+        position = position_dodge(width = 0.9),
+        size = 0.4,
+        shape = 21
+      )
```

Comment:
```suggestion
      ) + 
      geom_hline(yintercept = avg_gene_exp, type = "dashed")
      
```
You will want to add a dashed line here for the average mean gene expression of all genes to add to the plot. This suggestion implies that you first create a variable in the function and before plotting with `avg_gene_exp=colMeans(logcounts(normalized_sce))`.